enhancements/sig-policy/89-operator-policy-kind/README.md

```diff
@@ -0,0 +1,310 @@
+   the operator and its CRDs might be deleted, or nothing might be deleted.
+
+### Non-Goals
+
+1. The installation of OLM on managed clusters will not be automated by this enhancement.
+2. Management of "Operands" will not be included in this new type - they should be defined in
+   separate `ConfigurationPolicies`, possibly with a
+   [dependency](../74-order-policy-execution/README.md) on the `OperatorPolicy`.
+3. Management of `CatalogSources` will not be included in the new type - they should be defined in
+   separate `ConfigurationPolicies`.
+
+## Proposal
+
+### OperatorPolicy Syntax
+
+```yaml
+apiVersion: policy.open-cluster-management.io/v1beta1
+kind: OperatorPolicy
+metadata:
+  name: my-operator
+spec:
+  remediationAction: enforce # or inform
+  severity: medium
+  operatorGroup: # optional
+    name: my-operator-group
+    namespace: own-namespace
+    target:
+      namespaces:
+        - foo
+        - bar
+    #   selector: # One of selector or namespaces is allowed.
+    #     matchLabels: # or matchExpressions
+    #       mydomain.io/prod: "true"
+    serviceAccountName: my-og-account # optional
+  subscription:
+    channel: stable
+    name: my-operator
+    namespace: own-namespace
+    source: my-catalog
+    sourceNamespace: my-catalog-namespace
+    startingCSV: my-operator.v0.1.0 # optional
+    endingCSV: Any # or None or e.g. 'my-operator.v0.2.0`
+  requireLatestVersion: false
+  pruneOnDeletion:
+    - subscriptions
+    - clusterserviceversions
+    - installplans
+    - customresourcedefinitions
+    - apiservicedefinitions
+```
+
+The optional `spec.operatorGroup` section defines an
+[OperatorGroup](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_operatorgroups.yaml).
+If not specified, and if the operator supports this InstallMode, then an
+[AllNamespaces](https://olm.operatorframework.io/docs/advanced-tasks/operator-scoping-with-operatorgroups/#targetnamespaces-and-their-relationship-to-installmodes)
+OperatorGroup will be created.
+
+The `spec.subscription` section includes fields from the spec of an [OLM
+Subscription](https://github.com/operator-framework/api/blob/master/crds/operators.coreos.com_subscriptions.yaml),
+excluding `config` and `installPlanApproval`. The `endingCSV` field allows configuration of which
+InstallPlans will be approved by the controller when the policy is set to `enforce`.
+
+The `spec.requireLatestVersion` field configures whether the "latest" version of the operator
+defined in the catalog on this cluster "should" be installed. Note that if the "latest" version is
+greater than the allowed `endingCSV`, then that version will not be automatically approved even if
+the `remediationAction` is set to enforce: instead, the policy will be NonCompliant.
+
+The `pruneOnDeletion` field allows configuration of what will be removed when the policy is deleted.
+A "recommended" setting for most workloads would be `[subscriptions, clusterserviceversions]`,
```

Comment:
Is this the default?

Reply:
It probably should be... I'm still thinking about the best way to do that. Since I want it to be possible to configure nothing to be deleted, an empty list does not imply the "default" behavior. And filling it with an empty list, vs not specifying the field at all, feels too subtle of a difference - I think it could be confusing.